lib/open_food_network/order_cycle_management_report.rb

```diff
@@ -138,7 +138,7 @@ def customer_code(email)
     def sanitize_params(params)
       params[:q] ||= {}
       params[:q][:completed_at_gt] ||= Time.zone.today - DEFAULT_DATE_INTERVAL
-      params[:q][:completed_at_lt] ||= Time.zone.today
+      params[:q][:completed_at_lt] ||= Time.zone.today + 1.day
```

Comment:
ok but to be consistent this would be a constant as well

on top:
DEFAULT_DATE_START = Time.zone.today - 1.month 
DEFAULT_DATE_END = Time.zone.today + 1.day

and then here
      params[:q][:completed_at_gt] ||= DEFAULT_DATE_START
      params[:q][:completed_at_lt] ||= DEFAULT_DATE_END

Reply:
Yes, about being consistent.

But whoops! Please don't set `DEFAULT_DATE_START` and `_END` to constants. `Time.zone.today` would be evaluated only once - during deployment - and the default dates would not change until you deploy again.

You can change `DEFAULT_DATE_INTERVAL` to be a hash instead:

```ruby
DEFAULT_DATE_INTERVAL = { from: -1.month, to: 1.day }.freeze
```

Then `Time.zone.today + value` to both.

Reply:
you are right 🙈 

on top:
DEFAULT_DATE_INTERVAL = { from: -1.month, to: 1.day }.freeze

and then here
params[:q][:completed_at_gt] ||= Time.zone.today + DEFAULT_DATE_INTERVAL.from
params[:q][:completed_at_lt] ||= Time.zone.today + DEFAULT_DATE_INTERVAL.to

Reply:
I notice you corrected the syntax here @luisramos0 ;-)

Reply:
Thanks for all the help :heart: